Template/Tag/AddThisTag.web.js

```diff
@@ -0,0 +1,24 @@
+(function () {
+    return function (parameters, TagManager) {
+        this.fire = function () {
+            var siteId = parameters.get('AddThisPubId');
+            if (siteId) {
+                var script = document.createElement('script');
+                var s = document.getElementsByTagName('script')[0];
+                script.type = 'text/javascript';
+                script.src = '//s7.addthis.com/js/300/addthis_widget.js#pubid=' + siteId;
+                s.parentNode.insertBefore(script, s);
+
+                var parentSelector = parameters.get('AddThisParentSelector');
```

Comment:
I wonder should this maybe executed before inserting the script? (haven't tested it but wondering how the script above would know this DOM was inserted afterwards).  I presume so far it works because it'll take a while to load the script above.

Reply:
It definitely doesn't hurt doing it first.